benchmark_runner/benchmark_operator/workload_flavors/perf_ci/uperf/internal_data/uperf_vm_template.yaml

```diff
@@ -43,20 +46,20 @@ spec:
      nthrs:
        - {{ nthr1 }}
        - {{ nthr2 }}
+       - {{ nthr3 }}
      runtime: {{ runtime }}
      kind: vm
      server_vm:
        dedicatedcpuplacement: false
-       sockets: 1
-       cores: {{ server_cores }}
+       sockets: {{ server_sockets }}
+       cores: 1
        threads: 1
        image: kubevirt/fedora-cloud-container-disk-demo:latest # your image must've ethtool installed if enabling multiqueue
-       limits:
-         memory: {{ server_limits_memory_GB }}Gi
        requests:
-         memory: 10Mi
+         memory: {{ server_memory_GB }}Gi
+         cpus: {{ server_sockets }}
```

Comment:
Are you sure we can add it for VM ?

Reply:
Yes we want to add cpu requests for VMs so we ensure they get the proper amount of cgroup cpu.shares, but there is a typo, should be `cpu` not `cpus`, will fix that!